### ISSUE TITLE
Several User Annoyances/bugs

### ISSUE THREAD

**Issue by Pnin1204** [Loopring/dexwebapp#212]
I've been participating in the trading competition, and I've come to discover several "annoyances" that are somewhere between bugs and user experiences.

1. When trading with multiple windows open, often the swap pages are not displaying the swap values accurately. When scalping, this can be very frustrating. I frequently reload the entire page just be be sure. Even when the values are entered and altered on the swap page, it may not actually be the current exchange rate until I reload the page.

2. When trading, the time-out on your login can be so frustrating. If I'm actively trading, and the page stops responding suddenly, then I may lose an opportunity and that can really be annoying to have to reload the page, then unlock the page in order to have my balances update. Why not look at my last active state and decide if you lock the page, then? Rather than timing out in the middle of a trade.

3. The middle bar between the buys and the sells continues to visibly move/shift based on new orders being entered up and down the page. I'm on brave browser. This is frustrating because if I look away, then the entire page may move into all sells or all buys and I have to frequently scroll.

4. This is an actual bug.  When a swap is rejected because the spread or the rate isn't correct anymore, then there is a "successful swap" popup when the truth is that it was rejected. This has caught me several times unaware, already, thinking a swap has gone through when in fact it was rejected because of that display issue mentioned above.

**Comment by kenpole** [comment]
Yes, I've now lost several opportunities because the UI continues to lock me out of trading in the middle of trades. Especially when scalping, it's extremely annoying.

Another bug, though. I've noticed that frequently the Open Orders won't show, but it will in Order History and I can cancel a transaction from there, instead.
